wiki/Beatmap/de.md

```diff
@@ -0,0 +1,112 @@
+*Hauptartikel: [Beatmap-Kategorie](Category)*
+
+Eine eingereichte Beatmap gehört zu einer der folgenden Kategorien, die sich über die Zeit ändern können:
+
+- [Friedhof](Category#friedhof)
+- [Work in Progress / Ausstehend](Category#work-in-progress-und-ausstehend)
+- [Qualifiziert](Category#qualifiziert)
+- [Ranked](Category#ranked)
+- [Approved](Category#approved)
+- [Loved](Category#loved)
+
+Einige Kategorien haben [Einreichungskriterien](/wiki/Ranking_Criteria) und erlauben den Beatmaps, vorübergehende oder dauerhafte [Ranglisten](#ranglisten) zu haben. Die beliebteste Methode, dies zu erreichen, ist das [Beatmap-Ranking-Verfahren](/wiki/Beatmap_ranking_procedure), während die zweite Option ist, die Beatmap in die Kategorie [loved](Category#loved) zu verschieben.
+
+## Beatmaps herunterladen
+
+Beatmaps können über verschiedene Wege erlangt werden:
+
+### Beatmap-Auflistung
+
+Beatmaps aus unterschiedlichen Kategorien stehen auf der Webseite in der [Beatmap-Auflistung](https://osu.ppy.sh/beatmapsets) zur Verfügung. Sie können nach bestimmten Kriterien gefiltert (z. B. Beatmaps von [osu!taiko](/wiki/Game_mode/osu!taiko), die von einem bestimmten Mapper erstellt wurden) und dann einzeln heruntergeladen werden. [osu!supporter](/wiki/osu!supporter) haben Zugriff auf einen erweiterten Satz an Filterkriterien, wie beispielsweise die Möglichkeit, Beatmaps zu sehen, auf denen ein bestimmter Rang erreicht wurde.
+
+### osu!direct
+
+osu!direct ist die in das Spiel eingebaute Beatmap-Auflistung und wird für den schnellen Zugriff verwendet. Sie bietet eine Reihe an Features ähnlich der Auflistung auf der Webseite. Allerdings ist sie nur für [osu!supporter](/wiki/osu!supporter) verfügbar.
+
+### Beatmap-Pakete
+
+*Hauptartikel: [Beatmap-Pakete](Packs)*
+
+Beatmaps, die im selben Monat gerankt wurden oder dasselbe Thema verfolgen (z. B. derselbe Künstler), werden als Massen-Download in Archive gepackt, die Beatmap-Pakete genannt werden. Die Liste aller offiziellen Pakete ist auf der [Webseite](https://osu.ppy.sh/beatmaps/packs) einsehbar.
+
+### Inoffizielle Quellen
+
+Es gibt auch sekundäre Quellen für Beatmaps, die **vom osu!-Team weder überprüft noch betrieben werden**. Jedoch sind sie in den Foren verfügbar und von der Community geschätzt. Solche Quellen beinhalten:
+
+- Drittanbieter für Beatmap-Spiegel, die auf einer Plattform gehostet werden
+- Pakete und Zusammenstellungen, die von Spielern gemacht und via [P2P-Netzwerke](https://de.wikipedia.org/wiki/Peer-to-Peer) geteilt werden, wie z. B. BitTorrent.
+
+## Beatmaps und Community
+
+### Ranglisten
+
+osu!-Spieler benutzen in allen Spielmodi Beatmaps, um [global im Rang aufzusteigen](/wiki/Performance_points) und [gegeneinander anzutreten](/wiki/Ranking). osu! hat unterschiedliche Arten von beatmapspezifischen Ranglisten, die alle, außer der ersten, nur für [osu!supporter](/wiki/osu!supporter) zur Verfügung stehen:
+
+- Globale Ranglisten, die in der gesamten aktiven Spielerbasis funktionieren
```

Comment:
```suggestion
- Globale Ranglisten, die die gesamte aktive Spielerbasis verarbeiten
```

Every following bulletpoint describes what players are included in the leaderboard, not what players it "works for" as in who can view them, so I think this is better.

Reply:
That was a mistranslation on my part. Makes more sense 👍

Reply:
```suggestion
- Globale Ranglisten, die die gesamte aktive Spielerbasis miteinbeziehen
```
🤔

Reply:
👍